app/gui2/stories/mockSuggestionDatabaseStore.ts

```diff
@@ -0,0 +1,27 @@
+import { type SuggestionEntry, type SuggestionId } from '@/stores/suggestionDatabase/entry'
+import { defineStore } from 'pinia'
+import { reactive, ref } from 'vue'
+import mockDb from './mockSuggestions.json' assert { type: 'json' }
+
+export type SuggestionDb = Map<SuggestionId, SuggestionEntry>
+export const SuggestionDb = Map<SuggestionId, SuggestionEntry>
+
+export interface Group {
```

Comment:
Why do we define a separate interface for mocked groups? I see another one in `suggestionDatabase/index.ts` and it is different.

Reply:
this file became outdated after the merge, but yeah good point.
 originally i didn't want to import from the original module since its import path gets rewritten into this module, but that can be avoided by using a relative path